containers/SignUpFormContainer.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable */
+import React from 'react';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
```

Comment:
You need to use gql from 'react-apollo', not from 'graphql-tag'. You cannot pass eslint with this.
You can check ```/components/createform.js``` to see.